libcxx/modules/std.compat/ctime.inc

```diff
@@ -14,15 +14,17 @@ export {
 
   using ::timespec _LIBCPP_USING_IF_EXISTS;
   using ::tm _LIBCPP_USING_IF_EXISTS;
-
   using ::asctime _LIBCPP_USING_IF_EXISTS;
   using ::clock _LIBCPP_USING_IF_EXISTS;
+  using ::strftime _LIBCPP_USING_IF_EXISTS;
+
+#ifndef _LIBCPP_ABI_VCRUNTIME
```

Comment:
Are you simply leaving these functions missing when using UCRT?

Also, it seem that we should modify this line `libcxx/utils/libcxx/test/features.py` to enable the tests:
https://github.com/llvm/llvm-project/blob/968d38d1d7d9de2d5717457876bba2663b36f620/libcxx/utils/libcxx/test/features.py#L330

Reply:
The issue on `<ctime>` these functions is that UCRT incorrectly adds `static` to them, so we can't directly `export` them. At this time we might simply skip `export` for them to make the standard modules buildable. MSVC STL chooses to reimplement them in the `std` namespace as a workaround.

Reply:
> Also, it seem that we should modify this line `libcxx/utils/libcxx/test/features.py` to enable the tests:
> 
> https://github.com/llvm/llvm-project/blob/968d38d1d7d9de2d5717457876bba2663b36f620/libcxx/utils/libcxx/test/features.py#L330

Is just removing that line enough?

Reply:
I guess we can try this:

`or ("_WIN32" in compilerMacros(cfg) and "__MINGW32__" in compilerMacros(cfg) and not _mingwSupportsModules(cfg)) `

Reply:
> I guess we can try this:
> 
> `or ("_WIN32" in compilerMacros(cfg) and "__MINGW32__" in compilerMacros(cfg) and not _mingwSupportsModules(cfg)) `

that didn't work, no test files were generated for modules, I'll look into it...

Reply:
I'm not familiar with how the tests are run/generated yet, I'll see...

Reply:
> I guess we can try this:
> 
> `or ("_WIN32" in compilerMacros(cfg) and "__MINGW32__" in compilerMacros(cfg) and not _mingwSupportsModules(cfg)) `

This change did not work, no test files were generated for modules.